tests/aws/services/apigateway/test_apigateway_api.py

```diff
@@ -68,197 +68,37 @@ def delete_rest_api_retry(client, rest_api_id: str):
 
 
 @pytest.fixture
-def apigw_create_rest_api(aws_client):
+def apigw_create_rest_api(aws_client, aws_client_factory):
+    if is_aws_cloud():
+        client_config = (
+            Config(
```

Comment:
Yes, that's right! I think for now we've only been hit by that in APIGW, and I believe we maybe have retries disabled by default normally for some reasons I can't remember.

I think it could be good to keep it confined to APIGW right now, and as it's the only one having issues, only for create/delete, if there was a good reason to have retries disabled.
I think we could open a PR to create some fixtures/utils for that?